### DISCUSSION TITLE
DIY Embroidery Machine BOM questions

### DISCUSSION THREAD

**Discussion by mgn-norm** [inkstitch/inkstitch#3165] [General]
I've been working on buying the required materials to build the DIY Embroidery Machine, but I'm having trouble figuring out what 8mm GT2 belt(s) I need. I assume 8mm is the width, but 8mm does not seem like a common size. I've found 6, 9 and 10 but for whatever reason 8mm just doesn't seem to be available. would this 6mm work? https://www.amazon.com/Upgrade-Non-Slip-Version-Printer-Printers/dp/B08R93QQ8Z 

Also would replacing my balance wheel with a timing pulley work? I picked up a singer 15-90 for cheap that I restored and wasn't exactly sure how to drive it off the stepper motor I picked up. 

thanks

@jameskolme

**Comment by mgn-norm** [comment]
@moolight-seashell Can I get your motor mount and gears stl files if you don't mind? thanks in advance

**Reply by moolight-seashell** [reply]
my design was a bit crappy and not very well tested, but I was having fun starting from scratch so that's what I did.
I don't know what you mean by “the DIY Embroidery”, is there a ready-made model?

I only used 3d printer parts, so yes, I used 6 mm belts with typical 3d printer pulleys.

https://www.amazon.fr/Diyeeni-Distribution-limprimante-Remplacement-Synchronous/dp/B07RY2RZKD?nsdOptOutParam=true

if you're interested in my 3d files, I'll have a look, but I'm not sure I've kept everything.
these are freecad files so i can give you the source files or step/stl files

actually i use a commercial embrodery machine

**Reply by mgn-norm** [reply]
ah, I already figured out my other questions, I was just somewhat reusing this since I didn't have a good way of contacting you otherwise.

but that'd be awesome if you could provide those files.
